packages/shorebird_cli/lib/src/shorebird_config_mixin.dart

```diff
@@ -48,4 +49,48 @@ mixin ShorebirdConfigMixin on ShorebirdCommand {
     final yaml = file.readAsStringSync();
     return Pubspec.parse(yaml);
   }
+
+  ShorebirdYaml addShorebirdYamlToProject(String appId) {
+    File(
+      p.join(Directory.current.path, 'shorebird.yaml'),
+    ).writeAsStringSync('''
+# This file is used to configure the Shorebird CLI.
+# Learn more at https://shorebird.dev
+
+# This is the unique identifier assigned to your app.
+# It is used by your app to request the correct patches from the Shorebird servers.
+app_id: $appId
```

Comment:
Eventually we will want to add commented out versions of the other parameters this supports to make this file self-documenting. :)

Reply:
Yup agreed